packages/components/src/components/post-tab-header/post-tab-header.tsx

```diff
@@ -29,15 +29,9 @@ export class PostTabHeader {
   render() {
     return (
       <Host data-version={version}>
-        <li class="nav-item">
-          <a
-            aria-selected="false"
-            class="tab-title nav-link"
-            href="#"
-            id={this.tabId}
-            role="tab"
-          >
-            <slot/>
+        <li class="nav-item" role="tab">
```

Comment:
> According to the [accessibility documentation on tabs](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/), it seems that the role="tab" must remain on the interactive element.

I didn't find your statement on the page, but anyway I agree that it might make more sense.

About the role="presentation", I think in this case the ul and li are not making any sense if we assign a role="presentation" and that we should remove both.

Reply:
> I didn't find your statement on the page

In the "Keyboard Interaction" section, the `tab` element should be focused.



> About the role="presentation", I think in this case the ul and li are not making any sense if we assign a role="presentation" and that we should remove both.

They are necessary for proper styling, currently the tabs are not showing:
<img width="589" alt="Screenshot 2023-10-12 at 07 20 48" src="https://github.com/swisspost/design-system/assets/33580481/acf9cca6-9528-4b0a-9484-3882bfaa98e9">

Reply:
Sorry for the confusion, the styles were broken with another PR and just fixed with this one: https://github.com/swisspost/design-system/pull/2066

I updated this PR